src/lib/config.ts

```diff
@@ -30,7 +30,12 @@ export default class Config implements ConfigInterface {
 			return null;
 		}
 
-		return this.config.find(x => x.name.toUpperCase() === config.toUpperCase()).value;
+		let configValue = this.config.find(x => x.name.toUpperCase() === config.toUpperCase());
+		if (configValue != null) {
+			return configValue.value;
+		}
+
+		return null;
```

Comment:
Is there not a better way to do this? Like 

    return this.config.find(x => x.name.toUpperCase() === config.toUpperCase())?.value

Or something equivalent?

Reply:
chaining like that with question marks is not currently supported in Typescript, although it is [currently being proposed](https://github.com/tc39/proposal-optional-chaining).

Reply:
Ah yeah. [I knew I'd seen it somewhere...](https://www.youtube.com/watch?v=COzBv_Ces3g) Just forgot the context.